### PR TITLE
Added script for image retiral

### PR DESCRIPTION
**Pull request by Shilpi-Das1**
```
**Output:**
image_retiral.py -m image_retiral_manifest.yaml -r "us-south" -k <apiKey> -
i "crn:v1:bluemix:public:cloud-object-storage:global:a/65b64c1f1c29460e8c2e4bbfbd893c2c:8aeefa98-c07b-4d22-aa7a-1694374ae275::"

Cos configuration Key                     Value
Last Updated            Friday, October 16 2020 at 18:04:12
Default Region          us-south
Download Location       C:\Users\ShilpiDas\Downloads
CRN                     crn:v1:bluemix:public:cloud-object-storage:global:a/65b64c1f1c29460e8c2e4bbfbd893c2c:8aeefa98-c07b-4d22-aa7a-1694374ae275::
AccessKeyID
SecretAccessKey
Authentication Method   IAM
URL Style               VHost
Service Endpoint

Image: centos-82-mkumatag.ova.gz not copied FAILED
The specified key does not exist.


Copied centos-82-10072020.ova.gz from ocp4-images-bucket to ocp4-images-bucket-retired
Deleted centos-82-10072020.ova.gz from ocp4-images-bucket 
```

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/29#" title="Author self-approved">Shilpi-Das1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **sudeeshjohn**
You can assign the PR to them by writing `/assign @sudeeshjohn` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Shilpi-Das1** [comment]
@sudeeshjohn @mkumatag Please review the code changes

**Comment by bpradipt** [comment]
@sudeeshjohn @mkumatag please review it

File: scripts/images/image_retiral.py
Comment (mkumatag) [inline scripts/images/image_retiral.py:48]: I don't see out and err variables aren't used anywhere in this function scope!

File: scripts/images/image_retiral.py
Comment (mkumatag) [inline scripts/images/image_retiral.py:38]: and err and ret are not used in the function scope

File: scripts/images/image_retiral.py
Comment (mkumatag) [inline scripts/images/image_retiral.py:85]: no error check for the above 3 function calls

**Review by mkumatag — CHANGES_REQUESTED** [review]
overall looks okay except those error checks

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/29#" title="Author self-approved">Shilpi-Das1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **sudeeshjohn** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @sudeeshjohn` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Shilpi-Das1** [comment]
@mkumatag Addressed all review comments. Please review that

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/29#issuecomment-726542082" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/29#" title="Author self-approved">Shilpi-Das1</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**
